recipes/gmap/2016.09.23/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: gmap
+  version: "2016.09.23"
```

Comment:
To clarify: 2016-09-23 was previously the "default" version in bioconda. I just moved it to a specific dir.